src/modules/theme/components/header/header.component.ts

```diff
@@ -60,9 +61,13 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy {
   ) {
 
     const user = this.authenticationStore.getUserInfo();
+    const userRole = this.authenticationStore.getUserRole();
+    const userContext = this.authenticationStore.getUserContextInfo();
+    
     this.user = {
       displayName: user.displayName,
-      description: `Signed in as ${this.authenticationStore.getUserRole()}`
+      description: user.type === UserTypeEnum.ACCESSOR.toString() ? `Logged in as ${userRole} (${userContext.organisation?.organisationUnit.name})` : `Logged in as ${userRole}`,

```

Comment:
This logic would probably be better in a method of the store (as it compares the user type), but OK for now... Let's see what the future holds...